tests/test_stocks.py

```diff
@@ -98,3 +95,35 @@ def test_fill_quote_history(self):
         stock_data = Stock.objects.filter(id=stock_id)
         self.assertGreater(len(data), 0)
         self.assertEqual(len(stock_data), 1)
+
+    def test_fill_for_stocks_invalid_call(self):
+        """
+        Endpoint only works with GET
+        """
+        request = self.client.post('/stocks/fill/', follow=True, secure=True)
+        self.assertEqual(request.status_code, 405)
+
+    @mock.patch.object(
+        Fetcher,
+        'getHistorical',
+        mock.MagicMock(side_effect=[pd.DataFrame({
+            'Close': [1.76, 2.51],
+            'Date': ["2016-10-08", "2016-10-09"],
+        }),
+            pd.DataFrame({
+                'Close': [2.51, 2.53],
+                'Date': ["2016-10-09", "2016-10-10"],
+            }),
+            pd.DataFrame({
+                'Close': [2.55, 2.58],
+                'Date': ["2016-10-11", "2017-10-12"],
+            })])
+    )
+    @pytest.mark.django_db(transaction=True)
+    def test_fill_for_stocks_in_DB(self):
+        """
+        Filling data for stock
+        """
+        Stock(name="Facebook", ticker="FB").save()
+        request = self.client.get('/stocks/fill/', follow=True, secure=True)
```

Comment:
Testing with a client is relatively expensive. Better call the methods you test directly